### PR TITLE
Finished implementing the assert literal boolean check

### PR DESCRIPTION
**Pull request by VarS311**
PR for issue #47

### CODE REVIEW

**Comment by JimmyRojas** [comment]
it looks good to me..

File: apex-checks/src/main/resources/org/sonar/l10n/apex/rules/apex/A1024.html [hunk 1]
```diff
@@ -4,12 +4,11 @@
   -->
```
Comment (JimmyRojas) [inline apex-checks/src/main/resources/org/sonar/l10n/apex/rules/apex/A1024.html:4]: you should create a new html file for A1025 and not replace the current for A1024

Comment (VarS311) [inline apex-checks/src/main/resources/org/sonar/l10n/apex/rules/apex/A1024.html:4]: right, I did create a new html and my check has the code A1025 now, but I accidentally deleted the wrong message. Will fix it right away

File: apex-checks/src/main/resources/org/sonar/l10n/apex/rules/apex/A1024.html [hunk 2]
```diff
@@ -9,7 +9,6 @@
 
 <h2>Noncompliant Code Example</h2>
 <pre>
-@isTest
```
Comment (JimmyRojas) [inline apex-checks/src/main/resources/org/sonar/l10n/apex/rules/apex/A1024.html:12]: restore this A1024.html file content

File: apex-checks/src/test/java/org/fundacionjala/enforce/sonarqube/apex/checks/testrelated/AssertLiteralBooleanCheckTest.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import org.junit.Before;
+import org.junit.Test;
+import org.sonar.squidbridge.api.SourceFile;
+import org.sonar.squidbridge.checks.CheckMessagesVerifier;
+
+import java.io.File;
+
+import static org.fundacionjala.enforce.sonarqube.apex.ApexAstScanner.scanFile;
+
+
```
Comment (JimmyRojas) [inline apex-checks/src/test/java/org/fundacionjala/enforce/sonarqube/apex/checks/testrelated/AssertLiteralBooleanCheckTest.java:17]: remove this extra line

Comment (JimmyRojas) [inline apex-checks/src/test/java/org/fundacionjala/enforce/sonarqube/apex/checks/testrelated/AssertLiteralBooleanCheckTest.java:17]: code style

**Pull request merged**